sentry-core/src/performance.rs

```diff
@@ -1130,7 +1136,7 @@ impl Span {
 /// This currently only yields the `sentry-trace` header, but other headers
 /// may be added in the future.
 pub struct TraceHeadersIter {
-    sentry_trace: Option<String>,
+    pub(crate) sentry_trace: Option<String>,
```

Comment:
Since it looks like you are only using this field to create a new `TraceHeadersIter`, I'd suggest keeping the field private and adding a `pub(crate)` constructor (`pub(super)` might also be sufficient).

```suggestion
    sentry_trace: Option<String>,
```

The constructor could look like:

```rust
pub(super) fn new(sentry_trace: String) {
    Self { sentry_trace: Some(sentry_trace) }
}
```